flow/project.py

```diff
@@ -2717,11 +3041,54 @@ def _verify_group_compatibility(self, groups):
         return all(a.isdisjoint(b) for a in groups for b in groups if a != b)
 
     def _verify_aggregate_project(self, aggregate):
```

Comment:
I would like to see this function renamed to something like `_aggregate_is_in_project`.